### PR TITLE
Switch to new extension

### PR DESCRIPTION
**Pull request by Apogate**

### CODE REVIEW

**Comment by Apogate** [comment]
Replace the jQuery extension with a licensed ([BSD 3 clause](https://raw.githubusercontent.com/4chan/4chan-JS/master/LICENSE)) extension from [4chan's open source repo](https://github.com/4chan/4chan-js).

Since we have a similar html structure that lets us do this, this is here for a number of reasons
- Gives us one of the most feature rich front ends for open source imageboards currently available
- One file to maintain
- Multiple new features
- RePod doesn't have to pay for lawyers to chase license infringements anymore

the extension was rewritten to be portable regardless of site configuration/pathing changes

**Comment by RePod** [comment]
>Gives us one of the most feature rich front ends for open source imageboards currently available

🤔 Doesn't seem all that portable to me.

File: _core/postform.php [hunk 1]
```diff
@@ -110,12 +77,7 @@ function format($resno = null, $admin = false) {
...
-        
-		if ($admin) $temp = "<div id='adminForm' style='display:none; align:center;' />" . $temp . "</div>";
 		
         return $temp;
     }
-
-}
-
-?>
+}
```
Comment (RePod) [inline _core/postform.php:83]: 🤔

**Review by RePod — APPROVED** [review]
Don't expect me to maintain, bugfix, or expand this.
It's not my problem.

Also inconsistent tags. Arbitrary timing to care about it now, when we had zero issues before.

**Pull request merged**